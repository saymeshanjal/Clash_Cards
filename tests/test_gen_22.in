11
2 170 5
4 182 5
12 101 4
10 368 2
2 208 2
2 184 1
16 317 4
16 102 5
4 217 3
6 144 2
8 25 4
