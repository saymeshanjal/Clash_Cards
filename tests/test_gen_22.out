0 2 0
0 6 0
208 441 275
76 77 99
0 1 1
1 1 0
3809 5406 3655
4248 8622 0
1 5 0
5 6 9
8 42 20
